guides/source/engines.md

```diff
@@ -423,7 +423,7 @@ Finally, the assets for this resource are generated in two files:
 `app/assets/stylesheets/blorgh/articles.css`. You'll see how to use these a little
 later.
 
-You can see what the engine has so far by running `bin/rails db:migrate` at the root
+You can see what the engine has so far by running `rake db:migrate` at the root
```

Comment:
see the description, I had a question relating to this - engines by default don't have `bin/rake`, they only come with the rails binary in the bin directory